src/adapters/MeetingsJSONAdapter.js

```diff
@@ -384,6 +394,28 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return captureStream;
   }
 
+  /**
+   * Returns available media devices.
+   *
+   * @param {string} type String specifying the device type.
+   * @returns {Array} Array containing media devices
+   * @private
+   */
+  // eslint-disable-next-line class-methods-use-this
+  async getAvailableDevices(type) {
+    let devicesByType = null;
```

Comment:
it's ok to name this just `devices`, `devicesByType` would imply some kind of object where devices are indexed by their type.